app/decorators/email_message.rb

```diff
@@ -24,7 +24,7 @@ def initialize_text(mail)
   end
 
   def initialize_user(mail)
-    User.find_by(email: mail.from)
+    User.find_by_email(mail.from.first)
```

Comment:
Are you sure? `mail.from` would be an array and that's intented, as far as I can tell.